content/blog/blog-2020-07-30-introducing-operators-index.md

```diff
@@ -0,0 +1,85 @@
+By creating a pull request that adds or updates an entry, the referenced operator packages are validated as part of a CI job. This is similar to running `kubectl kudo package verify` on the operator package. Additional checks confirm that the metadata of the entry matches the respective metadata of the operator package.
+
+Once the pull request is approved and merged, it is added to the community repository.
+
+### Adding Tagged versions from a git repository
+
+Say you have an operator developed in a git repository. In this git repository you run tests, provide documentation and bundle the actual operator package. You use tags to release new versions of this operator.
+
+The following YAML would add an example operator "My Operator" from a GitHub repository to the index. The operator package is in the `operator` folder of this repository and a tag for version 1.0.0 exists.
+
+```yaml
+apiVersion: index.kudo.dev/v1alpha1
+kind: Operator
+name: My Operator
+gitSources:
+  - name: my-operator
+    url: https://github.com/example/my-operator.git
+versions:
+  - operatorVersion: "1.0.0"
+    git:
+      source: my-operator
+      directory: operator
+      tag: v1.0.0
+```
+
+[The entry for the KUDO Cassandra operator uses this approach.](https://github.com/kudobuilder/operators-index/blob/main/operators/cassandra.yaml)
+
+### Adding specific commits from a git repository
+
+Sometimes, one doesn't have tags for a specific operator. For example when multiple operators share a git repository. In this case, one can reference the SHA of a specific commit instead of a tag.
+
+```yaml
+apiVersion: index.kudo.dev/v1alpha1
+kind: Operator
+name: My Operator
+gitSources:
+  - name: my-operator
+    url: https://github.com/example/my-operator.git
+versions:
+  - operatorVersion: "1.0.0"
+    git:
+      source: my-operator
+      directory: operator
+      sha: 2b5b8ae28a83eb171d1c3094b92a1cc07e8c26f4
+```
+
+[The entry for the MySQL operator uses this approach.](https://github.com/kudobuilder/operators-index/blob/main/operators/mysql.yaml)
+
+### Adding tarballs
+
+If is also possible to reference a tarball of an operator package if there isn't a git repository. The tarball will be copied into the community repository once this entry is merged.
+
+```yaml
+apiVersion: index.kudo.dev/v1alpha1
+kind: Operator
+name: My Operator
+versions:
+  - operatorVersion: "1.0.0"
+    url: https://example.org/my-operator-1.0.0.tgz
+```
+
+## Summary
+
+The new [Operators Index](https://github.com/kudobuilder/operators-index) provides a one-to-one mapping of referenced operators and packages in the community repository. This makes it convenient for operator developers to add operator packages hosted in git repositories to KUDO's community repository.
```

Comment:
Maybe add a paragraph that operator projects can now start to provide a KUDO operator in addition to helm charts to give users the choice on what to use?

Reply:
Great idea!